projects/valtimo/task/src/lib/task-list/task-list.component.html

```diff
@@ -49,22 +49,27 @@
   </cds-tabs>
 </ng-template>
 
-<ng-template #caseLinkWithTaskLock let-data="data">
-  <a
-    class="float-left"
-    cdsLink
-    href="javascript:void(0)"
-    (click)="openRelatedCase($event, data.index)"
-  >
-    {{ 'task-list.goToCase' | translate }}
-  </a>
-  <div *ngIf="tasks[currentTaskType].tasks[data.index].isLocked">
-    <span
-      class="float-right badge badge-pill badge-secondary bg-grey"
-      ngbTooltip="{{ 'task-list.taskLocked' | translate }}"
+<ng-template #caseLink let-data="data">
+  <div *ngIf="tasks[currentTaskType].tasks[data.index].caseLocked">
+    <a
+      class="float-left cds--link--disabled"
+      cdsLink
+      href="javascript:void(0)"
+      ngbTooltip="{{ 'task-list.caseLocked' | translate }}"
+      (click)="$event.stopPropagation()"
+    >
+      {{ 'task-list.goToCase' | translate }}
+    </a>
+  </div>
+  <div *ngIf="!tasks[currentTaskType].tasks[data.index].caseLocked">
+    <a
+      class="float-left cds--link"
+      cdsLink
+      href="javascript:void(0)"
```

Comment:
Why is this here?

Reply:
To make sure the color of the link is correct